src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback } from 'react';
+import classNames from 'classnames';
+import { TodoStatus } from '../../types/TodoStatus';
+
+type Props = {
+  handleFilterBy: (str: TodoStatus) => void,
+  filterBy: TodoStatus,
+  itemsLeft: number,
+  completedTodos: number,
+  clearCompletedTodos: () => void,
+};
+
+export const TodoFooter: React.FC<Props> = React.memo(({
+  handleFilterBy,
+  filterBy,
+  itemsLeft,
+  completedTodos,
+  clearCompletedTodos,
+}) => {
+  const onClick = useCallback((event: React.MouseEvent<HTMLAnchorElement>) => {
+    const target = event.target as HTMLAnchorElement;
```

Comment:
Using innerText might lead to potential issues with whitespaces and other formatting. Consider using a different approach to get the text content.